### DISCUSSION TITLE
Whitespace between release metadata and notes

### DISCUSSION THREAD

**Discussion by ayushsharma82** [community/community#6498] [Other Feature Feedback, Questions, & Ideas] [answered]
Whitespace between the release metadata and notes make it hard to track the actual thing you are looking for. To be clear, The padding between metadata and notes should be decreased so that eyes won't have to move too much while scrolling down.

![image](https://user-images.githubusercontent.com/26592968/137806877-91170eb4-4529-4c2c-8f9a-a44b825a57cb.png)

**Comment by MylesBorins** [comment, marked as answer]
We just shipped an update to this, please take a look
